### DISCUSSION TITLE
How to broadcast message to subset of object group/collection?

### DISCUSSION THREAD

**Discussion by czajah** [DARMA-tasking/vt#2095] [Q&A] [answered]
How to avoid sending messages one by one and expensive serialization when I want to broadcast to chosen subset of nodes in object group?

Similar question goes for collection. For example if I have 2D collection how to broadcast to one column or row or set of columns or rows?

**Comment by lifflander** [comment, marked as answer]
Sorry, missed this. There isn't a good way to broadcast to just a column or row. You could create a group using `theGroupManager()` that spans the set of nodes required to target a row or column.
